website/content/docs/v0.7/Learn More/architecture.md

```diff
@@ -3,129 +3,17 @@ title: "Architecture"
 weight: 3
 ---
 
-In this section we will discuss the various components of which Talos is comprised.
+Talos is designed to be **monolithic** in _deployment_ and **modular** in _composition_.
```

Comment:
nit: I don't necessarily disagree but "monolithic" scares a lot of people. Might I suggest "atomic" instead? Not a blocker for me. Just a super nit against the negative connotation of the word "monolith" :smile: .